.github/workflows/ci.yml

```diff
@@ -20,6 +20,18 @@ env:
   RUSTFLAGS: -Dwarnings
 
 jobs:
+  # Test and fmt the npm pkg
+  npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v4
+      - name: npm fmt
+        with:
```

Comment:
I think this belongs in the step above

Reply:
hrm? the with tells node what version to use

Reply:
Sorry, I think at the time I submitted it this was attached to the step below. It's fine now!